pkgs/by-name/ba/bat/package.nix

```diff
@@ -38,7 +38,20 @@ rustPlatform.buildRustPackage rec {
       --prefix PATH : "${lib.makeBinPath [ less ]}"
   '';
 
-  checkFlags = [ "--skip=pager_more" "--skip=pager_most" ];
+  checkFlags = [
+    "--skip=alias_pager_disable_long_overrides_short"
+    "--skip=config_read_arguments_from_file"
+    "--skip=env_var_bat_paging"
+    "--skip=pager_arg_override_env_noconfig"
+    "--skip=pager_arg_override_env_withconfig"
+    "--skip=pager_basic"
+    "--skip=pager_basic_arg"
+    "--skip=pager_env_bat_pager_override_config"
+    "--skip=pager_env_pager_nooverride_config"
+    "--skip=pager_more"
+    "--skip=pager_most"
+    "--skip=pager_overwrite"
+  ];
```

Comment:
This is a lot of tests failing, perhaps that should be mentioned to upstream?

Reply:
These test cases are failing because they depends on `more` command as the `pager`.
We skipped the `parger_more` test case and `parger_most` test case in the past, so I skip the test cases that depends on `more`.
```
---- pager_most_from_bat_pager_env_var stdout ----
thread 'pager_most_from_bat_pager_env_var' panicked at 'Failed to spawn Command { cmd: "more", stdin: None, timeout: None }: No such file or directory (os error 2)', /build/bat-0.24.0-vendor.tar.gz/assert_cmd/src/cmd.rs:558:17

---- pager_most_from_pager_arg stdout ----
thread 'pager_most_from_pager_arg' panicked at 'Failed to spawn Command { cmd: "more", stdin: None, timeout: None }: No such file or directory (os error 2)', /build/bat-0.24.0-vendor.tar.gz/assert_cmd/src/cmd.rs:558:17

---- pager_most_from_pager_env_var stdout ----
thread 'pager_most_from_pager_env_var' panicked at 'Failed to spawn Command { cmd: "more", stdin: None, timeout: None }: No such file or directory (os error 2)', /build/bat-0.24.0-vendor.tar.gz/assert_cmd/src/cmd.rs:558:17

---- pager_most_with_arg stdout ----
thread 'pager_most_with_arg' panicked at 'Failed to spawn Command { cmd: "more", stdin: None, timeout: None }: No such file or directory (os error 2)', /build/bat-0.24.0-vendor.tar.gz/assert_cmd/src/cmd.rs:558:17

---- pager_overwrite stdout ----
thread 'pager_overwrite' panicked at 'Failed to spawn Command { cmd: "more", stdin: None, timeout: None }: No such file or directory (os error 2)', /build/bat-0.24.0-vendor.tar.gz/assert_cmd/src/cmd.rs:558:17


failures:
    alias_pager_disable_long_overrides_short
    config_read_arguments_from_file
    env_var_bat_paging
    pager_arg_override_env_noconfig
    pager_arg_override_env_withconfig
    pager_basic
    pager_basic_arg
    pager_env_bat_pager_override_config
    pager_env_pager_nooverride_config
    pager_more
    pager_most_from_bat_pager_env_var
    pager_most_from_pager_arg
    pager_most_from_pager_env_var
    pager_most_with_arg
    pager_overwrite
```

Reply:
then perhaps a comment about why would be helpful for posterity?

Reply:
@cafkafk 
The PR has been updated, please check it again, thanks.